client/js/components/button/BackToTopButton.vue

```diff
@@ -0,0 +1,79 @@
+<template>
+  <dp-button
+    :aria-label="Translator.trans('backToTop')"
+    :title="Translator.trans('backToTop')"
+    variant="outline"
+    color="primary"
+    :class="{ 'hide-visually': hide }"
+    hide-text
+    rounded
+    icon="arrow-up"
+    icon-size="large"
+    :style="buttonPosition"
+    @click="scrollTop">
+  </dp-button>
+</template>
+
+<script>
+import { DpButton } from '@demos-europe/demosplan-ui'
+
+export default {
+  name: 'DpBackToTop',
+
+  components: {
+    DpButton
+  },
+
+  data () {
+    return {
+      buttonPosition: '',
+      contentHeight: 0,
+      footerHeight: 0,
+      positionFromLeft: 0,
+      scrollPos: 0,
+      windowHeight: 0
+    }
+  },
+
+  computed: {
+    hide () {
+      return this.scrollPos < this.windowHeight
+    }
+  },
+
+  methods: {
+    calculateSizes () {
+      this.positionFromLeft = document.getElementById('jumpContent').offsetWidth + document.getElementById('jumpContent').offsetLeft
+      this.contentHeight = document.documentElement.scrollHeight - document.documentElement.offsetHeight
+      this.footerheight = document.querySelector('#app footer').offsetHeight
+      this.windowHeight = document.documentElement.clientHeight
+    },
+
+    calculatePosition () {
+      this.scrollPos = document.documentElement.scrollTop
+      const fromBottom = (this.contentHeight - 10 - this.footerheight > this.scrollPos) ? 10 : -this.contentHeight + 10 + this.footerheight + this.scrollPos
+
+      this.buttonPosition = `bottom: ${fromBottom}px; left: ${this.positionFromLeft}px; position: fixed`
+    },
+
+    scrollTop () {
+      window.scrollTo(0, 0)
+    }
+  },
+
+  mounted () {
+    this.calculateSizes()
+    this.calculatePosition()
+
+    window.addEventListener('resize', () => {
+      this.calculateSizes()
```

Comment:
`calculateSizes()` should also run if dimensions change in other ways (eg. if accordions are triggered). [ResizeObserver](https://developer.mozilla.org/en-US/docs/Web/API/Resize_Observer_API) is a good way to implement that. Also see scroller implementation: client/js/directives/scroller.js:58

Reply:
like so? 4459b273c89d8522f096db624c2098747e1600a2